### PR TITLE
updated vcorr

### PR DESCRIPTION
**Pull request by rcooke-ast**
Addresses #1462 - should this hotfix be directed to release?

### CODE REVIEW

**Review by profxj — APPROVED** [review]
Thanks, @rcooke-ast !

Looks like a unit test may still be failing...
But I am approving

File: pypeit/telescopes.py [hunk 1]
```diff
@@ -1,171 +1,193 @@
...
+                                              longitude=loc.lon.to(units.deg).value,
+                                              latitude=loc.lat.to(units.deg).value,
+                                              elevation=loc.height.to(units.m).value,
                                               diameter=4.2)
                                 
 class APFTelescopePar(TelescopePar):
     def __init__(self):
         super(APFTelescopePar, self).__init__(name='APF',
-                                              longitude=121.642778,
+                                              longitude=-121.642778,
```
Comment (kbwestfall) [inline pypeit/telescopes.py]: Can we use `loc = EarthLocation.of_site('Lick Observatory')`?

**Review by kbwestfall — APPROVED** [review]
Nice!  Thanks for the fix.  I checked and there is a "Lick Observatory" site.  Did you try that for APF?

Also, I think it would be fine if this goes to release, but does it carry with it many other changes from previous merges to `develop`?  If so, we should probably keep it as going to develop.

File: pypeit/telescopes.py (hunk 1 above)
Comment (rcooke-ast) [reply pypeit/telescopes.py]: Given that it's predominantly used by planet hunters (I presume), I didn't dare change it from the default value... I was being overly cautious, perhaps... Happy to update this...

**Comment by rcooke-ast** [comment]
OK - I've checked that this at least reduces kast data  :-)

Merging...

**Pull request merged**